configs/ignite.json

```diff
@@ -35,7 +35,7 @@
     },
     "pytorch-ignite-ai": {
       "lvl0": {
-        "selector": ".prose h1",
+        "selector": "nav > h5, .prose h1",
```

Comment:
Our scraper acts as `document.querySelectorAll(...)`, which in this case will select multiple `h1`s.

The best would be to add an `.active` class (or something similar) when a children is selected so we can target them directly